lib/WP_Auth0_Lock10_Options.php

```diff
@@ -193,6 +193,10 @@ public function get_lock_options() {
 
     $options_obj = array_replace_recursive( $extraOptions, $options_obj, $extended_settings );
 
+    if ( ! $this->wp_options->is_wp_registration_enabled() && ! isset( $options_obj["allowSignUp"] )) {
+      $options_obj["allowSignUp"] = false;
```

Comment:
what if `is_wp_registration_enabled` is false so that the first condition passes; but then the `allowSignup` is true. You would keep `allowSignUp=true` instead of turning it off (since `is_wp_registration_enabled` is false)

Reply:
Using the main pathway, yes, but some of that is hook-able so it can be overridden. These extra settings JSON fields say they will override all other settings and I want them to be able to do that, even if it's edge case